### PR TITLE
Sample project showing how to deploy an Akka Cluster with Apache Mesos 

### PR DESCRIPTION
**Pull request by ddragosd**
This is still WIP but I'm submitting it to get your feedback.
It illustrates a simple way to deploy an Akka Cluster in Mesos. 
I noticed this thread on the user list: https://groups.google.com/forum/#!topic/akka-user/zDdShzOddpU and I thought it would be great for the community to have something working in order to get started easier.

Thanks in advance for your feedback.

### CODE REVIEW

**Comment by ktoso** [comment]
Thanks, this is very cool! We'll need to figure out where this should live.
In a similar example for kubernetes recently https://github.com/akka/akka-samples/pull/34 we decided it to get it's own repo. I think this would be the right way for this example too... 

We'll get back to you here shortly, thanks a ton for the writeup!

**Comment by ddragosd** [comment]
thanks @ktoso . I'll wait for the new repo to be created then and I'll move this PR in there.

File: akka-sample-mesos-scala/.gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*#
+*.iml
+*.ipr
+*.iws
+*.pyc
```
Comment (raboof) [inline akka-sample-mesos-scala/.gitignore]: Hmm I'm not sure we expect `*.pyc` files here :) - perhaps you might want to add those to your [global gitignore](https://davidwalsh.name/global-gitignore)? OK by me thoughl

**Comment by raboof** [comment]
Did this get stuck on deciding whether it should live in its own repo? How does it relate to the demo's at https://github.com/akka/akka-management/tree/master/bootstrap-joining-demo ?

**Comment by ddragosd** [comment]
I wasn't aware of the repo you referred to. I'll check it out to see the marathon sample in there in more depth.

**Comment by raboof** [comment]
I think it'd be great to have a Mesos example, and I think it'd deserve to have its own repository.

If still interested, could you update this example to live in its own repo? Then after that I certainly think we transfer to to the Akka organization and could make sure it is featured on https://developer.lightbend.com/start/?group=akka .

I'll close this PR here for now, but am definitely interested in looking at this again when in its own repo.

**Pull request closed without merging**